pkg/trtl/config/config.go

```diff
@@ -36,12 +37,19 @@ type DatabaseConfig struct {
 }
 
 type ReplicaConfig struct {
-	Enabled        bool          `split_words:"true" default:"true"`
-	PID            uint64        `split_words:"true" required:"false"`
-	Region         string        `split_words:"true" required:"false"`
-	Name           string        `split_words:"true" required:"false"`
-	GossipInterval time.Duration `split_words:"true" default:"1m"`
-	GossipSigma    time.Duration `split_words:"true" default:"5s"`
+	Enabled        bool          `split_words:"true" default:"true" json:"enabled"`
+	PID            uint64        `split_words:"true" required:"false" json:"pid"`
+	Region         string        `split_words:"true" required:"false" json:"region"`
+	Name           string        `split_words:"true" required:"false" json:"name"`
+	GossipInterval time.Duration `split_words:"true" default:"1m" json:"gossip_interval"`
+	GossipSigma    time.Duration `split_words:"true" default:"5s" json:"gossip_sigma"`
+}
+
+type ReplicaStrategyConfig struct {
+	HostnamePID bool   `split_words:"true" default:"false"` // Set to true to use HostnamePID
```

Comment:
Should we consider making this default to true given that the present approach will default to using this strategy ?

Reply:
In a replicated trtl deployed as a StatefulSet, this has to be true - but for local development, single node trtl, and Docker compose this has to be false. I figured that right now at least, the last three use cases are more routine/normal.